modules/sdk-coin-ltc/test/unit/index.ts

```diff
@@ -1,45 +1,46 @@
 import 'should';
-import { getUtxoCoin } from '../util';
 
-describe('Custom LTC tests', function () {
-  const ltc = getUtxoCoin('ltc');
-  const tltc = getUtxoCoin('tltc');
+import { TestBitGoAPI, TestBitGo } from '@bitgo/sdk-test';
+
+import { Ltc, Tltc } from '../../src';
+import { BitGoAPI } from '@bitgo/sdk-api';
+
+describe('Litecoin:', function () {
+  const bitgo: TestBitGoAPI = TestBitGo.decorate(BitGoAPI, { env: 'test' });
+  bitgo.initializeTestVars();
+  bitgo.safeRegister('ltc', Ltc.createInstance);
+  bitgo.safeRegister('tltc', Tltc.createInstance);
+
+  const ltc = bitgo.coin('ltc') as Ltc;
+  const tltc = bitgo.coin('tltc') as Tltc;
```

Comment:
see comment here https://github.com/BitGo/BitGoJS/pull/2440/files#r912627038

Reply:
I did a rebase so I might have removed what was there (if I had changed it previously locally). `getUtxoCoin` is in `bitgo` right now while these tests are in `sdk-coin-ltc`. If we do any refactoring and consolidating on the tests side, method movements will probably happen when we do the utxo test refactor for all the utxo packages.